wof/WaterML_1_1.py

```diff
@@ -402,10 +402,11 @@ def quote_xml(inStr):
         pos = mo.end()
     s3 = s1[pos:]
     s2 += quote_xml_aux(s3)
-    return s2.decode(ExternalEncoding)
+    return s2
 
 
 def quote_xml_aux(inStr):
+    inStr = inStr.decode('utf-8')
```

Comment:
If I got the code correctly `inStr` comes from the data read, so here we must decode, work with strings, and encode at the end if necessary.